src/w3c/headers.js

```diff
@@ -634,11 +634,23 @@ export function run(conf, doc, cb) {
   conf.isPR = conf.specStatus === "PR";
   conf.isPER = conf.specStatus === "PER";
   conf.isMO = conf.specStatus === "MO";
+  conf.isNote = ["FPWD-NOTE", "WG-NOTE"].includes(conf.specStatus);
```

Comment:
@deniak, let me know if "FPWD-NOTE" applies or not. I'm was unsure.

Reply:
Yes, I think that applies too!